Dockerfile

```
@@ -1,72 +1,20 @@
-ENV NEXT_PUBLIC_FS_APP_ID=$NEXT_PUBLIC_FS_APP_ID
-ENV NEXT_PUBLIC_FS_MEASUREMENT_ID=$NEXT_PUBLIC_FS_MEASUREMENT_ID
-WORKDIR /app
-COPY --from=deps /app/node_modules ./node_modules
-COPY . .
-RUN yarn build
+FROM node:18-bookworm-slim
 
-# Production image, copy all the files and run next
-FROM node:16-alpine AS runner
-WORKDIR /app
+RUN apt-get update && apt-get install -y ca-certificates && npm install -g pm2
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)